### PR TITLE
refactor(models): Use Messages Raw model (2/N)

### PR DESCRIPTION
**Pull request by sampaiodiego**
## Issue(s)

ARCH-929

### CODE REVIEW

File: apps/meteor/client/methods/unpinMessage.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ Meteor.methods<ServerMethods>({
 			return 0;
 		}
 		dispatchToastMessage({ type: 'success', message: t('Message_has_been_unpinned') });
-		return ChatMessage.update(
+		ChatMessage.update(
```
Comment (KevLehman) [inline apps/meteor/client/methods/unpinMessage.ts:29]: needed?

File: apps/meteor/app/lib/server/functions/deleteMessage.ts [hunk 2]
```diff
@@ -63,7 +64,8 @@ export async function deleteMessage(message: IMessage, user: IUser): Promise<voi
 	Rooms.decreaseMessageCountById(message.rid, 1);
 
 	if (showDeletedStatus) {
-		Messages.setAsDeletedByIdAndUser(message._id, user);
+		// TODO is there a better way to tell TS "IUser[username]" is not undefined?
```
Comment (KevLehman) [inline apps/meteor/app/lib/server/functions/deleteMessage.ts:67]: NonNullable maybe, but's the same syntax lol

Comment (KevLehman) [reply apps/meteor/app/lib/server/functions/deleteMessage.ts:67]: There's also the ! (non null assertion) but it's banned in our rules

File: apps/meteor/client/methods/unpinMessage.ts (hunk 1 above)
Comment (sampaiodiego) [reply apps/meteor/client/methods/unpinMessage.ts:29]: yes.. this is client stub version that should reflect same as server version

**Review by ggazzo — APPROVED** [review]

**Pull request merged**